src/jade/run/benchmark.py

```
@@ -795,9 +778,74 @@ def _run_sub_benchmarks(
+        settings_mat_path = Path(
+            self.config.additional_settings_path, "MaterialsSettings.csv"
+        )
+
+        zaid_settings = pd.read_csv(settings_path, sep=",").set_index("Z")
+        settings_mat = pd.read_csv(settings_mat_path, sep=",").set_index("Symbol")
+
+        limit = self.config.custom_inp
+        if limit is not None and not isinstance(limit, int):
+            raise ConfigError(
+                "The custom input for the Sphere benchmark should be an integer"
+            )
```

Ruff (0.13.2)

Avoid specifying long messages outside the exception class

(TRY003)